spec/index.bs

```diff
@@ -163,6 +197,18 @@ spec: WebIDL; urlPrefix: https://heycam.github.io/webidl/#
   A <a>sync registration</a> has an associated <a>service worker registration</a>. It is initially set to null.
 
   Within one <a>list of sync registrations</a> each <a>sync registration</a> MUST have a unique <a>tag</a>.
+
+  A <dfn>periodic sync registration</dfn> is a tuple consisting of a <a>tag</a>, a dictionary of <a>options</a>, and a <a lt="registration state">state</a>.
+
+  A <a>periodic sync registration</a> has an associated <a>tag</a>, a DOMString.
+
+  A <a>periodic sync registration</a> has an associated <dfn>options</dfn>, a dictionary containing <dfn>minInterval</dfn>, a long long. <a>minInterval</a> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen.
```

Comment:
Why not just reference the IDL type when you introduce it? I'd add a TODO for that

Reply:
The spec is introducing these in layers -- first the concept of the two types of background sync registrations, and then what each consists of.

Reply:
Ok, in that case there is no need for the `options` dictionary, you can just define the `minInterval` associated with it

Reply:
why not define options however? If somebody comes across options later on in the spec, and not mininterval, they can be linked back to here. Also in the future we might have more than minInterval as part of options.

Reply:
It's an unnecessary abstraction if you don't want to use the IDL type directly. Just reference the flags a registration needs, clubbing them together here just adds more complexity